dev/com.ibm.ws.jaxws.ejb_fat/bnd.bnd

```diff
@@ -37,7 +37,14 @@ fat.project: true
 tested.features: \
   jaxws-2.2,\
   ejb-3.2, \
```

Comment:
ejb-3.2 was there before.  It would seem that ejb-3.2 isn't needed here.  Only ejbLite is used.

Reply:
Should jaxb-2.3 be added to this list?